src/plugins/intel_gpu/src/kernel_selector/cl_kernels/select_gpu_ref.cl

```diff
@@ -24,14 +29,19 @@ const uint d1  = (uint) get_global_id(0);
 const uint d2  = (uint) get_global_id(1);
 const uint d34 = (uint) get_global_id(2);
 
-const uint d3  = d34 % OUTPUT_SIZES[2];
-const uint d4  = d34 / OUTPUT_SIZES[2];
-
-uint output_offset = OUTPUT_OFFSET +
-                     d1*OUTPUT_PITCHES[0] +
-                     d2*OUTPUT_PITCHES[1] +
-                     d3*OUTPUT_PITCHES[2] +
-                     d4*OUTPUT_PITCHES[3];
+#ifdef IS_DYNAMIC
+    const uint d3 = d34 % OUTPUT_FEATURE_NUM;
+    const uint d4 = d34 / OUTPUT_FEATURE_NUM;
```

Comment:
Is it feature_num? Curious because the shape aware kernel uses output_sizes[2] :)

Reply:
Dims in *_sizes are stored in reverse order. So output_sizes[2] in xyfb order is feature_num